src/supervisor-api.ts

```diff
@@ -200,8 +204,15 @@ export class SupervisorAPI {
 			}
 		});
 
-		this.server = this.api.listen(port);
-		this.server.timeout = apiTimeout;
+		return new Promise(resolve => {
+			this.server = this.api.listen(port, () => {
+				log.debug(`Supervisor API successfully started on port ${port}`);
```

Comment:
```suggestion
				log.info(`Supervisor API successfully started on port ${port}`);
```

It's a small thing, but eventually debug output won't be shown by default, but IMO this should always be shown

Reply:
I've applied this change and followed this behavior with the message when supervisor shutdowns as well.